src/ActionList/Item.tsx

```diff
@@ -130,6 +145,21 @@ const StyledItem = styled.div<{variant: ItemProps['variant']; item?: ItemInput}
     }
   }
 
+  // Item dividers
+  :not(:first-of-type):not(${StyledDivider} + &):not(${StyledHeader} + &) {
+    margin-top: ${({showDivider}) => (showDivider ? `1px` : '0')};
+
+    ${StyledItemContent}::before {
+      content: ' ';
+      display: block;
+      position: relative;
+      top: -7px;
+      // NB: This 'get' won’t execute if it’s moved into the arrow function below.
+      border: 0 solid ${get('colors.selectMenu.borderSecondary')};
+      border-top-width: ${({showDivider}) => (showDivider ? `1px` : '0')};
```

Comment:
Can you elaborate on this?  I'm not familiar with the `NB:` acronym 😄

Reply:
To elaborate on “NB”—it’s short for [_“nota bene”_](https://en.wikipedia.org/wiki/Nota_bene), which is Latin for “note well.” The phrase is used to indicate extra attention or care should be paid to a remark.

To elaborate on the comment—I don’t know why interpolating the call to `get`, as in—
```js
border: ${({showDivider}) => showDivider ? `1px solid ${get('colors.selectMenu.borderSecondary')}` : '0')};
```
—doesn’t work, but it doesn’t: If used, no `border` style is applied to the pseudoelement (other styles—`content`, `display`, `position`, `top`—(without interpolated calls to `get`) are applied)